### ISSUE TITLE
Add music tracks

### ISSUE THREAD

**Issue by shoc71** [shoc71/dont-stop-now#227]
This issue won't be resolved until;

- there are 5 tracks for the main branch
- 3 tracks exclusive for the secret branch (+5 track from main)
- 1 tracks exclusively for the good branch  (+5 track from main)
- 1 tracks exclusively for the bad branch (+5 track from main)

**Comment by slysnivy** [comment]
Would the tracks for each branch be played at random?

**Comment by shoc71** [comment]
likely yes, the songs would be played at random, except for the last hardest levels where they get their own track.

**Comment by slysnivy** [comment]
@shoc71 I think this is outdated too lol

**Comment by shoc71** [comment]
> @shoc71 I think this is outdated too lol

Agreed. WAYY too outdated, but the objective for having more music in the game remains pretty much the same.
